client/src/components/relatedItems/RelatedItemsWidget.jsx

```diff
@@ -37,25 +43,72 @@ class RelatedItemsWidget extends Component {
       relatedProductsIds: [],
       loaded: false
     })
+
     this.setOverviewIdData();
   }
 
-
   setOverviewIdData() {
-
     this.setState({
       loaded: false
     })
-
     const endpoints = [
       `http://localhost:8080/products/${this.state.overviewId}`,
```

Comment:
Once you deploy, you will need to change your endpoints to the following:
`'/products/${this.state.overviewId}'`
The reason behind this is that you would get an error on the deployed instance saying it doesnt know what localhost is. You can do this now as well. These changes will work on a deployed state and local state.